go/vt/vtctl/reparentutil/planned_reparenter_flaky_test.go

```diff
@@ -1641,6 +1641,10 @@ func TestPlannedReparenter_performGracefulPromotion(t *testing.T) {
 				ctx = _ctx
 			}
 
+			durability, err := GetDurabilityPolicy("none")
+			require.NoError(t, err)
+			tt.opts.durability = durability
+
```

Comment:
Do we have any unit tests for the other policies? All the tests in this file seem to be using "none".
If not, it will be good to add some (a follow up PR is fine for that).

Reply:
Planned reparent shard isn't using the durability policy for any revocation or establishment logic. It is only using the policies for knowing who sends semi-sync acks, but that is something none of the tests have, so any durability policy is fine for these tests. But, I'll add one more test which tests semi-sync based on the durability-policy being used in a follow up PR!

Reply:
As promised, this test for checking that the correct value of semi-sync is sent from PRS has been added - https://github.com/vitessio/vitess/pull/10425